app/assets/javascripts/hub.coffee

```diff
@@ -17,9 +17,11 @@ $(document).on 'turbolinks:load', ->
   #-----------------------
   if window.location.hash
     hash = window.location.hash.substring(1)
-    $('html, body').animate {
-      scrollTop: ($('a[name="'+hash+'"]').offset().top) + 15
```

Comment:
@rushenn What's the root cause of this issue? Please explain this line in detail. What is being done to scrollTop?